guides/15_Background_Filter_Video_Processor.md

```diff
@@ -0,0 +1,145 @@
+
+### Content delivery, caching and bandwidth
+
+Background blur model files are loaded from an Amazon Content Delivery Network (CDN) at runtime. This provides low-latency global distribution without the need to serve a full suite of files as part of your application.
+
+Model files will be cached indefinitely by end users’ browsers, so that subsequent uses of background blur on the same device will take less time to initialize.
+
+In addition to having network connectivity to [Amazon Chime’s media services, as described in the documentation](https://docs.aws.amazon.com/chime/latest/dg/chime-components.html), the use of background blur requires access to Amazon CloudFront via HTTPS (port 443). All requests will be to subdomains of `sdkassets.chime.aws`. End users whose network configurations prevent access to this CDN, or applications that do not include the correct domain in Content Security Policy (see below), will fail support checks and be unable to use background blur.
+
+CloudFront’s IP address ranges are documented in the [Amazon CloudFront Developer Guide](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/LocationsOfEdgeServers.html).
+
+The overhead of loading model files can add latency to parts of your application. The browser cache will make subsequent loads significantly faster. Check for support and create background blur resources at appropriate times to minimize the impact of this latency: for example, during a lobby or device picker interaction.
+
+## Preparing your application
+
+### Content Security Policy
+
+Modern web applications use [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to protect users from certain classes of attacks. You will need to include the following in your policy to allow the Amazon Chime SDK access to resources it needs at runtime:
+
+* `script-src` and `script-src-elem`: add `https://*.sdkassets.chime.aws` to load video processing code.
+* `connect-src`: add `https://*.sdkassets.chime.aws` to load model files via `fetch`.
+* `worker-src`: add `blob:` to load worker JavaScript across origins.
+* `child-src`: add `blob:` to load worker JavaScript across origins (only in Safari).
+
+If you omit any of these entries, or if you use both HTTP headers and `http-equiv` `meta` tags to specify policy and inadvertently exclude any of these by intersection, then background blur will not be able to initialize, and will either appear to be unsupported or will create a no-op vide frame processor. You will see errors in your browser console like:
+
+```
+Refused to connect to
+'https://static.sdkassets.chime.aws/bgblur/workers/worker.js…'
+because it violates the document's Content Security Policy.
+```
+
+If you need guidance with correct CSP configuration to use background blur, contact AWS Support.
```

Comment:
VF has some issue with Chrome 95+ and need to update the CSP guidance, do we have to do the same thing for background blur?

Reply:
This does apply to background blur as well as they both use the CDN to download assets at runtime.